### PR TITLE
solrconfig.xml should live within curation_concerns-models

### LINKED ISSUE #153: Missing solrconfig.xml when running generator.
```
     warning  GENERATING CURATION_CONCERNS FULL-TEXT
Could not find "/Users/justin/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/solr_conf/conf/solrconfig.xml" in any of your source paths. Your current source paths are:
/Users/justin/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/curation_concerns-models-0.1.0/lib/generators/curation_concerns/models/templates
```

### PR DESCRIPTION
**Pull request by tampakis**
Closes #153.

### CODE REVIEW

**Comment by jcoyne** [comment]
Shouldn't this go in the templates directory?

**Comment by tampakis** [comment]
Yeah that makes more sense. I'll update the PR.

**Comment by tampakis** [comment]
@jcoyne - I moved the solrconfig.xml with the other config templates.

File: curation_concerns-models/lib/generators/curation_concerns/models/fulltext_generator.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ def banner
   # Copy CurationConcerns's solrconfig into the dir from which the jetty:config task pulls
   # CurationConcerns's solrconfig includes full-text extraction
   def copy_solr_config
-    src = File.join(__FILE__, '..', '..', '..', '..', '..', '..', 'solr_conf', 'conf', 'solrconfig.xml')
+    src = File.join(__FILE__, '..', 'templates', 'config', 'solrconfig.xml')
     copy_file File.expand_path(src), 'solr_conf/conf/solrconfig.xml', force: true
```
Comment (jcoyne) [inline curation_concerns-models/lib/generators/curation_concerns/models/fulltext_generator.rb]: No need to do this expand path anymore, just `copy_file 'config/solrconfig.xml', 'solr_conf/conf/solrconfig.xml', force: true`

Comment (tampakis) [inline curation_concerns-models/lib/generators/curation_concerns/models/fulltext_generator.rb]: Yep.

**Comment by mjgiarlo** [comment]
:+1:

**Pull request merged**